definitions/tools/add_strelka_gt.cwl

```diff
@@ -88,6 +109,12 @@ requirements:
             sub parse_gt {
                 my ($gt_str, $ids) = @_;
                 my @gt_ids = map{$ids->{$_}}(split //, $gt_str);
+                my $i;
+                foreach $i (@gt_ids) {
```

Comment:
If you don't need it to persist outside of the loop, it might be better to declare the  loop variable directly in the for loop, like: 
```perl
for my $i (@gt_ids) {
```
(`for` and `foreach` are the same so whichever you prefer 😄)

Reply:
I had tried that earlier and for some reason it kept failing on me which is why I pulled it out. I can double back and try it again but that's how I landed here